frontend/components/providers/posthog-provider.tsx

```
@@ -0,0 +1,141 @@
+export function PostHogProvider({
+  children,
+  userId,
+  userEmail,
+  userName,
+}: PostHogProviderProps) {
+  const pathname = usePathname();
+  const searchParams = useSearchParams();
+
+  useEffect(() => {
+    // Initialize PostHog (only once)
+    if (typeof window !== "undefined" && !(window as any).posthog) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)